packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/react/DevMenuAwareReactNativeHost.kt

```diff
@@ -0,0 +1,17 @@
+package expo.modules.devmenu.react
+
+import android.app.Application
+import android.os.Bundle
+import com.facebook.react.ReactInstanceManager
+import com.facebook.react.ReactNativeHost
+import expo.modules.devmenu.protocoles.DevMenuDelegateProtocol
+
+/**
+ * Basic [ReactNativeHost] which knows about expo-dev-menu and implements [DevMenuDelegateProtocol].
+ */
+abstract class DevMenuAwareReactNativeHost(application: Application)
+  : ReactNativeHost(application), DevMenuDelegateProtocol {
+  override fun appInfo(): Bundle? = null
+
+  override fun reactInstanceManager(): ReactInstanceManager = super.getReactInstanceManager()
```

Comment:
```suggestion
```
We don't need it here, do we? 🤔

Reply:
That is funny :D 
We don't need this but we should add it. Why? Cause Android Studio doesn't parse it correctly. 
Maybe because we have the same method in the `ReactNativeHost` and in `DevMenuDelegateProtocol`.  So when you want to go to method body, the android studio doesn't find it :/